index.html

```diff
@@ -1,24 +1,58 @@
 <!DOCTYPE html>
 <html lang="en">
 	<head>
+		<link href='https://fonts.googleapis.com/css?family=Open+Sans:400,300,300italic,400italic,600,800' rel='stylesheet' type='text/css'> 
     	<meta charset="utf-8">
-    	<meta name="description" content="Your description goes here">
-    	<meta name="keywords" content="one,two,three">
+    	<meta name="description" content="Relaxr: Get piece of mind with a single tap">
+    	<meta name="keywords" content="Startup,Relaxr,New York">
 
-		<title>Blank Template</title>
+		<title>Relaxr</title>
 
 		<!-- CSS links -->
-
+		<link rel="stylesheet" type="text/css" href="css/normalize.css">
+		<link rel="stylesheet" type="text/css" href="css/style.css">
 	</head>
 	<body>
-		<div>
-			<header>
-				<h1>Blank Template</h1>
-				<h2>Just another blank template</h2>
+	<div class="container">
+			<header class="clearfix">
+					<h1>Relaxr</h1>
+					<div class="slogan">
+						<p>Get piece of mind with a single tap</p>
+					<button class="button1">Get it Now</button>
+					</div>
 			</header>
-			<footer>
-				&copy; Footer Goes Here
-			</footer>
-		</div>
-	</body>
+			<main class="clearfix">
+				<article class="benefits">
+				<h2>Benefits</h2>
+				<p>The perfect personal assistant. Relaxr does your job for you so you can enjoy life as it is meant to be.</p>
+					<ul>
+						<li>Schedule meetings for you</li>
+						<li>Excel automation to complete your work for you</li>
+						<li>Responds to emails on your behalf</li>
+						<li>Does all your work for you with our revolutionary AI technology</li>
+					</ul>
+				</article>
+				<article class="testimonial">
+						<p>“Relaxr changed my life. I’ve been able to travel the world, spend limited time working and my boss keeps thanking me for crushing work."</p>
+						<p>&mdash;Amanda, Intuit</p>
+				</article>		
+				<article class="sign-up">
+						<button class="button2">Sign Up Now!
+						</button>
+				</article>
+				<article class="social">
+					<h2>Relaxr</h2>
+					<table>
```

Comment:
your probably don't need a table here. You could have just done something like this:

```
<div class="social-buttons">
 <img src="images/twitter.png" alt="twitter">
 <img src="images/facebook.png" alt="facebook">
</div>
```

```
.social-buttons {
 text-align: center;
}
```